src/ol/layer/WebGLTile.js

```diff
@@ -474,7 +477,7 @@ class WebGLTileLayer extends BaseTileLayer {
 
   /**
    * Update any variables used by the layer style and trigger a re-render.
-   * @param {Object<string, number>} variables Variables to update.
+   * @param {Object<string, (string|number)>} variables Variables to update.
```

Comment:
Previously, the initial variables allowed strings and numbers as values, which is also the documented behavior, but `updateStyleVariables()` only allowed numbers. Now both accept strings and numbers.

Reply:
@tschaub, is this desired? Or should it be the other way around, i.e. only accept numbers everywhere?

Reply:
Nevermind. I'm removing this change. A quick glance on how the `stringLiteralsMap` works, it looks like variable updates really only work with numbers.